packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -823,7 +824,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
       };
     }
 
-    if (items && items.length === 0) {
+    if (items && (items.length === 0 || items.length === value.length)) {
```

Comment:
Please could you add a test for that new case?

Reply:
Done